shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -850,6 +859,149 @@ \subsection{Verifiable Random Function}
   \label{fig:rules:vrf}
 \end{figure}
 
+\clear
+
+\subsection{Decentralization Slider}
+\label{sec:decentralization-slider}
+
+The transition from the boostrap era to a fully decentralized network is explained in
+section 3.9.2 of \cite{delegation_design}.
```

Comment:
maybe make it a non-breaking space

Reply:
I don't see what looks weird. It's breaking after the "3.9.2" and looks normal to me. Am I not seeing something?

Reply:
doesn't look weird, but non-breaking space would make the citation glue to the `of`, not allowing a line-break between them. But certainly not necessary, just a `chktex` warning.

Reply:
ah! that makes sense, thanks